app/src/main/java/com/example/sharingang/MapFragment.kt

```diff
@@ -5,6 +5,7 @@ import android.annotation.SuppressLint
 import android.location.Location
 import android.os.Bundle
 import android.os.Looper
+import android.util.Log
```

Comment:
You can remove it if you don't use it I guess.

Reply:
Good point!